app_dart/lib/src/request_handling/regular_request_handler.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:meta/meta.dart';
+
+import '../datastore/cocoon_config.dart';
+import 'body.dart';
+import 'exceptions.dart';
+import 'request_handler.dart';
+
+/// A [RequestHandler] that handles regular requests.
```

Comment:
What's a "regular" request?

How is it different from the api handler we have already?

Reply:
regular request differs from API handler on needing no authentication.

Reply:
Then the name should reflect the fact, consider something like "NoAuthRequestHandler".

I wonder why this request doesn't need auth? Time series data is public?

Reply:
Changed to NoAuthRequestHandler as suggested to make it clear.

Yes, they are public.